src/reuse/modules/mobile/userInteraction.ts

```diff
@@ -61,7 +63,126 @@ export class UserInteraction {
         vl.log("Checkbox already selected.");
       }
     } catch (error) {
-      this.ErrorHandler.logException(error);
+      this.ErrorHandler.logException(error, "Error: During check", true);
+    }
+  }
+
+  /**
+   * @function doubleTap
+   * @memberof mobile.userInteraction
+   * @description Double Tap's on the mobile element.
+   * @param {Element | string} element - The element or CSS selector describing the element (e.g., accessibility ID, XPath, CSS,).
+   * @param {number} [timeout = 30000] - The timeout to wait(ms) (default: 30000).
+   * @example
+   * const elem = await mobile.userInteraction.doubleTap(elem);
+   * const elem = await mobile.userInteraction.doubleTap(elem, 20000);
+   */
+  async doubleTap(element: Element, timeout: number = parseFloat(process.env.QMATE_CUSTOM_TIMEOUT!) || 30000) {
+    const vl = this.vlf.initLog(this.doubleTap);
+
+    try {
+      vl.log("Expecting element to be displayed and enabled");
+      await Promise.all([
+        expect(element).toBeDisplayed({
+          wait: timeout,
+          interval: 100,
+          message: `Timeout '${+timeout / 1000}s' by waiting for element is displayed.`
+        }),
+        expect(element).toBeEnabled({
+          wait: timeout,
+          interval: 100,
+          message: `Timeout '${+timeout / 1000}s' by waiting for element is enabled.`
+        })
+      ]);
+      vl.log("Double taping on the element");
+      await element.doubleClick();
+      vl.log("Given element is successfully double taped on the mobile Ui");
+    } catch (error) {
+      this.ErrorHandler.logException(error, "Error: During doubleTap", true);
+    }
+  }
+
+  /**
+   * @function setText
+   * @memberof mobile.userInteraction
+   * @description Enter a string value into a mobile input box.
+   * @param {Element | string} element - The selector for the input element (e.g., accessibility ID, XPath, CSS,).
+   * @param {string} value - The string value to be entered.
+   * @param {boolean} clear - Whether to clear the input box before entering the value (default: true).
+   * @param {number} [timeout = 30000] - The timeout to wait (ms).
+   * @returns {Promise<void>}
+   * @example
+   * const elem = await mobile.userInteraction.setText(elem);
+   * const elem = await mobile.userInteraction.setText(elem, 2000, false);
+   */
+  async setText(element: Element, value: string, timeout: number = parseFloat(process.env.QMATE_CUSTOM_TIMEOUT!) || 30000, clear: boolean = true): Promise<void> {
```

Comment:
to keep consistency with other namespaces, we should name it `fill` or `clearAndFill`

Reply:
I like the option to specify if the function should clear the input as well or not but to keep consistency, I would prefer to add separate function as we did for ui5 and nonUi5 namespaces. 
- clear
- clearAndFill
- fill

Reply:
If you split the functions as Ben said, you won't need the `clear`boolean parameter in the end

Reply:
I have added a new methods which Ben suggested, now we have all these 3 methods at mobile side.
```
fill()
clearAndFill()
clear()
```